lib/terraform/runner/response.rb

```diff
@@ -0,0 +1,56 @@
+require 'json'
+
+module Terraform
+  class Runner
+    class Response
+      include Vmdb::Logging
+
+      attr_reader :stack_id, :stack_name, :status, :action, :message, :error_message,
+                  :details, :created_at, :stack_job_start_time, :stack_job_end_time
+
+      # @return [String] Extracted attributes from the JSON response body object
+      def self.parsed_response(http_response)
+        data = JSON.parse(http_response.body)
+        _log.debug("data : #{data}")
+        Terraform::Runner::Response.new(
+          :stack_id             => data['stack_id'],
+          :stack_name           => data['stack_name'],
+          :status               => data['status'],
```

Comment:
since we're checking for upcased SUCCESS/FAILED, should we do:

```suggestion
          :status               => data.fetch('status', '').upcase,
```